komga-webui/src/views/BrowseBook.vue

```diff
@@ -233,6 +233,23 @@
         </v-row>
       </template>
 
+      <!--  Library    -->
+      <v-row class="align-center text-caption">
+        <v-col cols="4" sm="3" md="2" xl="1" class="py-1 text-uppercase">{{ $t('common.library') }}</v-col>
+        <v-col cols="8" sm="9" md="10" xl="11" class="py-1">
+          <v-chip
+            :class="$vuetify.rtl ? 'ml-2' : 'mr-2'"
+            :title="getLibraryName(series)"
```

Comment:
shouldn't that be `getLibraryName(book)` ?

Reply:
Yep fixed